### ISSUE TITLE
Strip Poker Professional rev. B (1995)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#3796]
Smiling Spectre — 08/04/2022
Strip Poker Professional rev. B can use the addon disks from the Strip Poker Professional (1994). Actually, known 3 data disks for rev.B contains exactly the same pictures as 1994 version, only rehashed differently.
Fast way to install everything is copy files directly. Each pack consist of:
- OPPSDD[x].TXT and OPPSDD1.VXL for the pack description and picture.
- [person].TXT, [person]S dir and [person].VXL for each different opponent.

Basically, only three persons that aren't in packs (because they are the base game) are Laura,Kami and Greta. So you simply must copy OPPSDD pairs, and all persons except this three.

**Comment by exoscoriae** [comment]
done
